internal/util/merr/errors.go

```diff
@@ -98,6 +98,9 @@ var (
 	// Metrics related
 	ErrMetricNotFound = newMilvusError("MetricNotFound", 1200, false)
 
+	// Allocator related
+	ErrIDAllocatorFailed = newMilvusError("IDAllocatorFailed", 1300, false)
```

Comment:
We don't need to add an error here, as failed to allocate a ID/timestamp is not a leaf error, the cause should be returned by the allocator itself